main.go

```diff
@@ -121,14 +122,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	useCombinedTokens, err := strconv.ParseBool(os.Getenv("USE_COMBINED_TOKENS"))
+	if err != nil {
+		useCombinedTokens = false
+	}
```

Comment:
Why is this option an environment variable instead of a flag?
Also, it would be good to differentiate between "didn't set `USE_COMBINED_TOKEN`" and "set an invalid value" and throw an error in the second case instead of quietly continuing.